SDL/Hit.cuh

```diff
@@ -85,7 +85,7 @@ namespace SDL
       return x - n * float(2.f * float(M_PI));
     }
     CUDA_HOSTDEV inline float phi(float x, float y) {
-      return phi_mpi_pi(float(M_PI) + atan2f(y,x));
+      return phi_mpi_pi(float(M_PI) + atan2f(-y,-x));
```

Comment:
is `phi_mpi_pi` even needed here? I missed the reason why 32c54ff278f9686674bb35bce92ca1512602e4f5 was applied here originally. Why are we computing [0, 2π] range first so that it can be immediately converted to [-π,π] ?

Reply:
I think 32c544ff commit written a phi_mpi_pi is a roll back of commit #296 61a75dc (under the same PR). Before the PR 296 of changing dPhi, the original written way was "phi_mpi_pi(float(M_PI) + Atan2f(-y,-x));"  I meant to follow the previous way in Hit.cuh, but somehow didn't do it correctly.
I also think it's weird that the range goes to [0,2pi] first, since the atan2f() function returns [-pi,pi] directly..

Reply:
I changed the function into "return ata2f(y,x)" and get the fake rate plot as this.
http://uaf-10.t2.ucsd.edu/~yagu/SDL_GPU_plots/fix_dphi/simpler_phi_PU200_NEVT-1_aed0d8D-PU200/mtv/var/TC_fakerate_etazoom.pdf
Compared to Manos's fake rate eta plots after fix, they are consistent with each other. Seems it's safe to use the simpler one

Reply:
Please do not add any update at this point. New updates should go in the `master` branch, after it is migrated to alpaka.

Reply:
> Compared to Manos's fake rate eta plots after fix, they are consistent

a comparison plot would be better, identical results are expected (within limits of reproducibility)